### DISCUSSION TITLE
How can I get values of analogue triggers?

### DISCUSSION THREAD

**Discussion by eliasdaler** [hajimehoshi/ebiten#1955] [Q&A] [answered]
There's no `ebiten.StandardGamepadAxisLeftTrigger` and `ebiten.StandardGamepadAxisRightTrigger`. 
How can I get the value of how much the analogue trigger was pressed?

**Comment by hajimehoshi** [comment, marked as answer]
See https://pkg.go.dev/github.com/hajimehoshi/ebiten/v2#StandardGamepadButtonValue. Probably this is what you need. 😄

**Reply by eliasdaler** [reply]
Thanks a lot, this did the trick!

```go
leftTriggerValue := ebiten.StandardGamepadButtonValue(gamepadID, ebiten.StandardGamepadButtonFrontBottomLeft)
rightTriggerValue := ebiten.StandardGamepadButtonValue(gamepadID, ebiten.StandardGamepadButtonFrontBottomRight)
```
